spec/game_spec.rb

```diff
@@ -1,20 +1,48 @@
 require 'game'
 
 describe Game do
-  it 'should provide the total score of a frame correctly' do
-    game = Game.new(:roll_one, :roll_two)
-    allow(game).to receive(:roll_one) { 3 }
-    allow(game).to receive(:roll_two) { 6 }
-    expect(game.total_of_frame).to eq(9)
+
+  describe '.initialize' do
+    it 'sets @rolls to an empty array' do
+      game = Game.new
+      expect(game.rolls).to eq([])
+    end
   end
 
-  it 'should determine when a spare is rolled' do
-    game = Game.new(6, 4)
-    expect(game.spare?).to be true
+  describe '#score' do
+    it 'scores the game as 0 when only gutter balls are thrown' do
+      game = Game.new
+      20.times { game.roll(0) }
+      expect(game.score).to eq(0)
+    end
   end
 
```

Comment:
Testing is good, nice to see how you changed them as they are a lot easier to read and understand